### PR TITLE
Tweak document settings to improve appearance and readability

### LINKED ISSUE #3645: Tweak doc appearance
### Description

Tweak `TextOptions so text is not blurry when zoomed in.

Set properties to optimise whitespace/layout for readability.

### PR DESCRIPTION
**Pull request by duncanp-sonar**
Fixes #3645

### CODE REVIEW

**Comment by duncanp-sonar** [comment]
These two flags are supposed to make documents easier to read.
The first modifies the whitespace to make justified text easier to read, and the second allows words at the end of lines to be hyphenated.

#### Without the flags
image

#### With the flags
image

The whitespace differences are subtle but apparent, but you have to zoom in to notice the hyphenation:
image

File: src/Education/Commands/RuleHelpUserControl.xaml [hunk 1]
```diff
@@ -2,20 +2,25 @@
...
+
+            <!-- Optimise whitespace and word-breaks for readability -->
+            <Setter Property="IsOptimalParagraphEnabled" Value="True" />
+            <Setter Property="IsHyphenationEnabled" Value="True" />
+
+            <!-- Reduce blurriness of font outlines. -->
+            <!-- Note: same settings as for the Help About box. Also, changes to these properties when
+                 debugging are not picked up immediately - you need to recompile to see the difference. -->
+            <Setter Property="TextOptions.TextRenderingMode" Value="ClearType" />
+            <Setter Property="TextOptions.TextFormattingMode" Value="Ideal" />
```
Comment (duncanp-sonar) [inline src/Education/Commands/RuleHelpUserControl.xaml:23]: @bigfluffycookie thanks for the info about which `TextOptions` to use.
Setting the `TextOptions` does make a difference, but you have to recompile to see it (unlike other properties, which you can change on the fly when debugging and see the difference immediately).

#### Without
image


#### With
image

File: src/Education/Commands/RuleHelpUserControl.xaml [hunk 2]
```diff
@@ -29,7 +34,6 @@
         </Style>
         <Style x:Key="Blockquote_Section" TargetType="Section">
             <Setter Property="Margin" Value="12,0,0,0"/>
-            <!--<Setter Property="Foreground" Value="{DynamicResource {x:Static vsshell:VsBrushes.CaptionTextKey}}"/>-->
```
Comment (duncanp-sonar) [inline src/Education/Commands/RuleHelpUserControl.xaml:32]: I deleted all of the commented-out setters.

**Review by bigfluffycookie — APPROVED** [review]
Awesome!

**Pull request merged**